core/src/stack/mod.rs

```diff
@@ -26,3 +31,80 @@ pub const B1_COL_IDX: usize = STACK_TOP_RANGE.end + 1;
 /// Index of the h0 helper column in the stack trace. This column contains 1 / (b0 - 16) when
 /// b0 != 16, and ZERO otherwise.
 pub const H0_COL_IDX: usize = STACK_TOP_RANGE.end + 2;
+
+// STACK INPUTS
+// ================================================================================================
+
+/// Initial state of the stack to support program execution.
+///
+/// The program execution expects the inputs to be a stack on the VM, and it will be stored in
+/// reversed order on this struct.
+#[derive(Clone, Debug, Default)]
+pub struct StackInputs {
+    values: Vec<Felt>,
+}
```

Comment:
If we are putting `StackInputs` here, should we also put `StackOutputs` here as well?

Reply:
Makes sense; done!